packages/desktopjs-electron/src/electron.ts

```diff
@@ -371,25 +384,40 @@ export class ElectronContainer extends WebContainerBase {
                     });
                 });
             }
-        } catch (e) {
-            // eslint-disable-next-line no-console
-            console.error(e);
-        }
 
-        let replaceNotificationApi = ElectronContainer.replaceNotificationApi;
-        if (options && typeof options.replaceNotificationApi !== "undefined") {
-            replaceNotificationApi = options.replaceNotificationApi;
-        }
+            if (options && options.autoStartOnLogin) {
+                this.electron.setLoginItemSettings({
+                    openAtLogin: options.autoStartOnLogin
+                });
+            }
 
-        if (replaceNotificationApi) {
-            this.registerNotificationsApi();
-        }
+            let replaceNotificationApi = ElectronContainer.replaceNotificationApi;
+            if (options && typeof options.replaceNotificationApi !== "undefined") {
+                replaceNotificationApi = options.replaceNotificationApi;
+            }
 
-        this.nodeIntegration = (options && options.node != null) ? options.node : null;
+            if (replaceNotificationApi) {
+                this.registerNotificationsApi();
+            }
 
-        this.screen = new ElectronDisplayManager(this.electron);
+            if (options && options.node) {
+                this.nodeIntegration = options.node;
+            }
+        } catch (e) {
+            // eslint-disable-next-line no-console
+            console.error(e);
+        }
+    }
 
-        this.globalShortcut = new ElectronGlobalShortcutManager(this.electron);
+    public isAutoStartEnabledAtLogin(): Promise<boolean> {
```

Comment:
I thought this was moved into a more generic getOptions so that we wouldn't have to hoist each option onto the container interface.  Did that get reverted or did we lose track of it/

Reply:
no. we didnt revert it. if we move to getOptions which is more suited to a specific window and not the entire app, i dont think it will a good fit.
Do you think otherwise ?

Reply:
> getOptions which is more suited to a specific window and not the entire app, i dont think it will a good fit.

ContainerWindow has a getOptions, the proposal here is adding getOptions to container which is a pair of setOptions.  What you set is returned by get.  This allows the flexibility of supporting any option without having to add properties.

Reply:
ah okay. Got it. yes. this makes sense. I will make the change

Reply:
Added getOptions method to containers. for now, returning only `autoStartOnLogin` but we can add as many options there. created a new interface as well `IContainerOptions` for this.

Reply:
Longer term I suppose we persist the options provided to setOptions and getOptions will return current property values applied on top of that?

Reply:
This works for now.  I'll merge but I'm probably going to flip getOptions to any and remove the interface.  That will alow flexibility in anything provided in the future on setOptions to be easily flipped back around on getOptions w/o type issues.